### PR TITLE
[Feature] Date customisation for issue #107

### LINKED ISSUE #107: [Proposal] Date format customisation
Provide an option to customise date format in _config.yml.

### PR DESCRIPTION
**Pull request by strauhmanis**
Closes #107

### CODE REVIEW

**Comment by daviddarnes** [comment]
This is a nice addition, thanks @strauhmanis! I'm not sure about the `site.alembic` prefix. As much as I like people using my theme, I don't want the configuration to stop them switching to another theme. Having an Alembic specific key in the config is going to leave legacy code for people. Can it be just `date_format`?

File: _sass/_theme.scss [hunk 1]
```diff
@@ -59,11 +59,11 @@ body {
 .logo {
   display: inline-block;
   line-height: 0;
-  max-height: 4rem;
+  height: 4rem;
```
Comment (daviddarnes) [inline _sass/_theme.scss]: Could this fix be in it's own pull request? It's a better way to track fixes. Also, might be better to style the image so the size isn't a fixed width? Some users may upload quite a wide but short image, in which this case it will never be bigger than `4rem`

**Comment by strauhmanis** [comment]
Tried to create a separate pull request by creating a new branch, but it did not work. Need to learn to do that!

**Comment by daviddarnes** [comment]
@strauhmanis I think if you create a new branch from master and make those svg changes you can open a new pull request, you can do that with the branch dropdown on your fork
screen shot 2018-09-21 at 13 28 45

**Comment by strauhmanis** [comment]
There are two pull request now, but it looks kinda messy :-)

**Comment by daviddarnes** [comment]
@strauhmanis hey! I updated both your pull requests so the changes you're proposing are in their own pull requests. Check out #110 and #108, they are small changes but now they are split into their own updates. You'll of course get credit for both!

**Comment by strauhmanis** [comment]
Hey! Glad that everything worked out in the end! I’ll practice with multiple branching before making several PR’s next time!

**Comment by daviddarnes** [comment]
Not to worry! Just me being picky. Really appreciate you contributing to the repo, thank you
